src/encoded/static/components/genome_browser.js

```diff
@@ -147,29 +146,48 @@ function mapGenome(inputAssembly) {
 /**
  * Display a label for a file’s track.
  */
-const TrackLabel = ({ file, long }) => {
-    const biologicalReplicates = file.biological_replicates && file.biological_replicates.join(',');
+const TrackLabel = ({ file, label, long }) => {
+    const biologicalReplicates = file.biological_replicates && file.biological_replicates.join(', ');
     return (
         <React.Fragment>
-            <a href={file['@id']} className="gb-accession">{file.accession}<span className="sr-only">{`Details for file ${file.accession}`}</span></a>
-            <ul className="gb-info">
-                {file.biosample_ontology && file.biosample_ontology.term_name ? <li>{file.biosample_ontology.term_name}</li> : null}
-                {file.target ? <li>{file.target.label}</li> : null}
-                {file.assay_term_name ? <li>{file.assay_term_name}</li> : null}
-                {long ?
-                    <React.Fragment>
-                        <li>{file.output_type}</li>
-                        {(biologicalReplicates !== '') ? <li>{`rep ${biologicalReplicates}`}</li> : null}
-                    </React.Fragment>
-                : null}
-            </ul>
+            {(label === 'cart') ?
+                <ul className="gb-info">
+                    {file.target ? <span>{file.target.label}, </span> : null}
+                    {file.assay_term_name ? <span>{file.assay_term_name}, </span> : null}
+                    {file.biosample_ontology && file.biosample_ontology.term_name ? <span>{file.biosample_ontology.term_name}</span> : null}
+                    {long ?
+                        <React.Fragment>
+                            <li><a href={file['@id']} className="gb-accession">{file.accession}<span className="sr-only">{`Details for file ${file.accession}`}</span></a></li>
```

Comment:
Oh, sorry, that was my mistake in the original code. We’re not supposed to ever use file.accession. We’re supposed to use file.title. Usually they’re the same, but on occasion file.accession is undefined, while file.title has the real identifier. I keep forgetting about that. I think file is the only accessioned object where you can’t use the accession property.

Reply:
Just for the cart, right?

Reply:
Not supposed to use file.accession anywhere. We’ve had several bugs in the past about this, though this happens only in unusual cases where we use lab IDs instead of accessions.